tests/integration/manifests/api-gateway.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: operator.kyma-project.io/v1alpha1
```

Comment:
could you move this manifests folder under `testsuites/gateway` like other testsuites ?